tapir/coop/templates/coop/shareowner_list.html

```diff
@@ -29,31 +29,37 @@ <h5>{% translate "Cooperative Members" %}</h5>
                     {% endfor %}
                 </div>
             </div>
-        </div>
-        <ul class="list-group list-group-flush" id="share_owner_table">
+        </div>   
+        <p style="text-align:right" class="m-3">
+        <button class="{% tapir_button_link_to_action %}" type="button" data-bs-toggle="collapse" data-bs-target="#collapseFilter" aria-expanded="false" aria-controls="collapseFilter">
+            Filters
```

Comment:
What about translation of that word to German?

Reply:
It would be "Filter" without "s". So maybe just keep it like this? Everybody should understand it.

Reply:
Not sure if I like this approach. Would that mean that from now on we don't need to use translations if there is one letter difference between English and German words?